src/index.js

```diff
@@ -271,18 +336,16 @@ class CollectUserData {
       `${DATA_FOLDER}/${ARTIFACT_FILE_NAME}.json`,
       JSON.stringify(json)
     );
-    await writeFileAsync(
-      `${DATA_FOLDER}/${ARTIFACT_FILE_NAME}.csv`,
-      csv
-    );
+    await writeFileAsync(`${DATA_FOLDER}/${ARTIFACT_FILE_NAME}.csv`, csv);
 
     await this.createandUploadArtifacts();
     await this.postResultsToIssue(csv);
     process.exit();
   }
 
-  normalizeResult() {
+  async normalizeResult() {
```

Comment:
Any reason you made this an `async` method? I can't spot any asynchronous in the method's code.

Reply:
Thanks for catching that. I made it async when I put the code to handle the SAML identities inside `normalizeResult()`. I ended up moving it out, but forgot to remove it.